build.gradle

```diff
@@ -41,7 +41,9 @@ dependencies {
     testRuntime 'org.objenesis:objenesis:2.6'
 }
 
-sourceCompatibility = 1.8
+allprojects {
+    sourceCompatibility = 1.8
```

Comment:
We want gradle profiler to be able to measure builds running on Java 8. Therefore at least the subprojects need to be compiled with Java 8 bytecode, since they run as init scripts on the build JVM. It seems straightforward to also make it possible to run Gradle profiler itself on Java 8, so you don't need to have two JDKs installed.